Documentation/Metrics/arangodb_process_statistics_major_page_faults_total.yaml

```diff
@@ -1,7 +1,7 @@
 name: arangodb_process_statistics_major_page_faults_total
 introducedIn: "3.6"
-help: |
-  Number of major page faults.
+help:
```

Comment:
The bar was there intentionally, since we wanted the line breaks to survive right until we have created the actual documentation source in the docs repository. Can you please readd them? Without the bars, the YAML spec says that multiple lines are concatenated and we end up with rather long lines.

Reply:
This is mostly relevant for `description` and `threshold` but for consistency, let's keep them for `help`, too.

Reply:
Thanks, I misinterpreted the output in `allMetrics.yaml`. Changed it back.